### PR TITLE
feat: init

### PR DESCRIPTION
**Pull request by alex-reiff**
### Description

Initial migration of codebase from GHE

### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

File: .one-pipeline.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
+version: '1'
```
Comment (ocofaigh) [inline .one-pipeline.yaml:1]: this wont be needed - we will need to add a github action pipeline

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
+# Changelog
```
Comment (ocofaigh) [inline CHANGELOG.md:1]: I would remove this file - we don't maintain changelogs currently

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,43 @@
+# -*- mode:makefile; coding:utf-8 -*-
```
Comment (ocofaigh) [inline Makefile:1]: many of the commands in this Makefile are no longer needed - please review

File: README.md [hunk 4]
```diff
@@ -1 +1,275 @@
...
+This is a companion GUI tool for [IBM Secure Landing Zone](https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone), primarily for deployments creating through the [IBM Cloud Catalog](https://cloud.ibm.com/catalog/architecture/deploy-arch-ibm-slz-vpc-9fc0fa64-27af-4fed-9dce-47b3640ba739-global).
+
+It can help create an override.json file used for configuring your deployment of Secure Landing Zone. It can also be used to modify or validate existing override.json files.
+
+
+Live Version: https://ibm.biz/override-landing-zone
+
+Direct Link: https://setup.deploy-arch-ibm-slz-vpc.cloud.ibm.com
+
+---
```
Comment (ocofaigh) [inline README.md:12]: Lets make it clear that the below headings are all for local developer set up and not for consumers

File: client/.editorconfig [hunk 5]
```diff
@@ -0,0 +1,14 @@
+# top-most EditorConfig file
```
Comment (ocofaigh) [inline client/.editorconfig:1]: what is using this?

File: client/package.json [hunk 6]
```diff
@@ -0,0 +1,55 @@
+{
+  "name": "slz-gui",
+  "version": "0.8.0",
+  "private": true,
+  "type": "module",
+  "repository": "git@github.com:carbon-design-system/carbon-tutorial.git",
+  "bugs": "https://github.com/carbon-design-system/carbon-tutorial/issues",
+  "license": "Apache-2.0",
+  "main": "/src/index.js",
+  "homepage": "https://pages.github.ibm.com/Alexander-Reiff/slzgui",
```
Comment (ocofaigh) [inline client/package.json:10]: needs to be updated

File: .gitmodules [hunk 7]
```diff
@@ -0,0 +1,3 @@
+[submodule "common-dev-assets"]
+	path = common-dev-assets
+	url = https://github.ibm.com/GoldenEye/common-dev-assets.git
```
Comment (ocofaigh) [inline .gitmodules:3]: this needs to be updated to the public repo

**Review by ocofaigh — CHANGES_REQUESTED** [review]
I left some comments. We need a pipeline that will run the tests here, but maybe we add a followup item for that.

File: client/package.json (hunk 6 above)
Comment (alex-reiff) [reply client/package.json:10]: Can you create the initial github page? I don't have access to the repo settings.

**Review by ocofaigh — APPROVED** [review]

**Pull request merged**